### ISSUE TITLE
Impact of Ledoit-Wolf covariance estimation

### ISSUE THREAD

**Issue by alexander-myltsev** [Marigold/universal-portfolios#41]
Hi @Marigold,

You suggested to implement Ledoit-Wolf covariance estimation at the end of [modern-portfolio-theory.ipynb](https://github.com/Marigold/universal-portfolios/blob/master/modern-portfolio-theory.ipynb). 

I read the paper. The shrinkage ratio of `F` estimator should go to zero when there are 11 assets or so. To show the benefit they made an estimation for ~900 assets having monthly historical data. I wonder what impact do you expect to achieve by implementing the method at the universal-portfolios?

**Comment by Marigold** [comment]
You're absolutely correct. I tried Ledoit-Wolf and the impact is minimal on a universe on ~10 assets. It might even screw up your optimization if you have assets with ~1 correlation in there (e.g. QQQ and TQQQ). There's even a cooler paper https://papers.ssrn.com/sol3/papers.cfm?abstract_id=3530390 with a similar, theoretically sound idea, but its impact is negligible too. The most critical thing is to get your mean estimates right... or at least not too wrong which is where Black-litterman or other regularization approaches comes to play.

**Comment by alexander-myltsev** [comment]
OK. Thanks for your reply.
Did you try Black-Litterman? What's the impact?

**Comment by alexander-myltsev** [comment]
@Marigold would you answer ⬆️ ?

**Comment by Marigold** [comment]
I haven't implemented it, BL is just a framework, you still need to come up with return estimates. I'm for instance using an assumption of constant Sharpe ratio across all assets (i.e. return = sharpe * volatility) and then put it into MPT optimization. That gives reasonable allocation across assets (I believe it even has a name, just don't remember what it was). Pretty much all experiments where I estimated expected returns failed.
